packages/apps/kadena-docs/src/pages/docs/build/guides/pact-local-api-queries.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Pact Local API queries
+description: Pact Local API queries
+menu: Build
+label: Pact Local API queries
+order: 2
+layout: full
+---
```

Comment:
Just for the record. Extra metadata we could consider adding in a later stage.

- keywords
- version
- dates